### DISCUSSION TITLE
Stuck at Welcome to Github Education screen

### DISCUSSION THREAD

**Discussion by messard1** [community/community#88618] [GitHub Education]
### Hi everyone,

![Screenshot 2024-01-17 170451](https://github.com/community/community/assets/116312277/f121eb45-33ff-4f18-8a77-d5528f538819)

Can't get past this screen. The links do nothing. Tried incognito and turning off ad blocker.

**Comment by Mostafa-Ayman1** [comment]
same problem :( and I don`t what should I do
